driver/node/opera.go

```diff
@@ -207,7 +207,18 @@ func (n *OperaNode) StreamLog() (io.ReadCloser, error) {
 }
 
 func (n *OperaNode) Stop() error {
-	return n.host.Stop()
+	// SigInt repeatedly up to 9 times
+	if err := network.Retry(9, 5*time.Second, func() error {
+		if err := n.Interrupt(); err == nil {
+			return fmt.Errorf("failed to interrupt node %w", err)
+		} else {
+			return nil
+		}
+	}); err == nil {
+		return n.host.Stop()
```

Comment:
I think the container should be stop all the time - if it was not stopped gracefully, what is purpose of keeping it running?

Reply:
Change removed.